packages/slice-machine/src/theme.ts

```diff
@@ -37,6 +37,10 @@ const AppTheme = (): Theme =>
           bg: "#F1EEFE",
           color: "#5842C3",
         },
+        deleted: {
+          bg: "rgba(203, 36, 49, 0.1)",
+          color: "#CB2431",
```

Comment:
We already have this colour, it's called "danger"

Reply:
Fixed